cpp/dolfinx/fem/DirichletBC.h

```diff
@@ -132,6 +132,71 @@ template <typename T>
 class DirichletBC
 {
 private:
+  /// @brief Build list of cells (local to process, including ghosts)
```

Comment:
I don't think we need 'local to the process'. It's implicit throughout the code.